src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -3109,6 +3109,46 @@ def __call__(self, *_inputs: T_OTXDataEntity) -> T_OTXDataEntity | None:
         return inputs
 
 
+class GetBBoxCenterScale(tvt_v2.Transform):
```

Comment:
The reason why this transform is added again is that the `BboxInfo` has related properties based on an image and it must be changed to one based on bounding box for TopDown, right?

Reply:
That's correct!